features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/include/tfm_mailbox.h

```diff
@@ -135,7 +135,7 @@ struct mailbox_reply_t {
 struct ns_mailbox_slot_t {
     struct mailbox_msg_t   msg;
     struct mailbox_reply_t reply;
-    const void             *owner;          /* Handle of the owner task of this
+    void                   *sem;            /* Handle of the semaphore of this
```

Comment:
Will this change be available in upstream TF-M?

Reply:
No

Reply:
I don't think we can accept changes here that will never be upstreamed to TF-M. We will update TF-M in the future and this would overwrite these changes.

Reply:
Jaeden, we are going to run into problems here as there are more PSA targets supported. I doubt all PSA targets will be on the same version of TFM.  E.g. the musca target is free to move along but that doesn't mean this target will move with it.

Reply:
Yes, this will be a problem if Mbed OS needs to support non-upstream versions of TF-M. Mbed OS can really only support upstream versions of TF-M, one version for all targets.

Reply:
This is not just non-upstream versions but different upstream versions.  There is TFM v1 and v1.1 today. Although the PSA API has unlikely changed, the exports likely have.  How will mbed os deal with different upstream versions?  Would we really force all PSA targets to conform to current mbedos version of TFM?  I think we need to have a discussion on how we handle all these cases.

Reply:
Reverted